tests/test_geometry.py

```diff
@@ -309,3 +312,25 @@ def test_shortest_torus_path():
                     break
             assert unseen_neg_minimiseds == set(), \
                 (start, end, width, height, neg_minimiseds)
+
+
+def test_standard_system_dimensions():
+    # Special case: 0
+    assert standard_system_dimensions(0) == (0, 0)
+
+    # Should crash on non-multiples of 3
+    with pytest.raises(ValueError):
+        standard_system_dimensions(1)
```

Comment:
Maybe... Though a one board system doesn't really have 8x8 chips ;). That said it seems like a reasonable expectation to have! Will add :)